plugins/modules/df_service.py

```diff
@@ -35,17 +36,17 @@
 options:
   env_crn:
     description: 
-        - The CRN of the CDP Environment to host the Dataflow Service
-        - Required when state is present
+      - The CRN of the CDP Environment to host the Dataflow Service
+      - The environment name can also be provided, instead of the CRN
+      - Required when state is present
```

Comment:
Can use 'Required when I(state=present)' to add a bit of formatting "flair."

Reply:
Consider it done.